app/src/main/res/layout/activity_backup_picker.xml

```diff
@@ -103,6 +103,18 @@
                 app:checked="@={prefs[`backup-automatic-enabled`]}"
                 app:showIfTrueAnimatedFastOff="@{vm.showAuto}" />
 
+            <CheckBox
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:checked="true"
```

Comment:
I think this checked = true could interfere with the one which follows the preference

Reply:
What do you mean by "the one which follows the preference"?

Reply:
I thought that's how I can have it enabled by default.

Reply:
Do you mean "False" in this?
Pref.getBooleanDefaultFalse(PREF_AUTO_BACKUP_WIFI)